src/parse_expr.jl

```diff
@@ -4,13 +4,8 @@
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 function _try_parse_idx_set(arg::Expr)
-    # The parsing of `x[i=1]` changed from `i=1; getindex(x, 1)` to
-    # `getindex(x; i=1)` from Julia v0.7 to Julia v1 so we need `:kw` for
-    # Julia v1.0 and :(=) for Julia v0.7
-    # The parsing of `[i=1]` is however still the same way so we need :(=) for
-    # both
-    is_julia_v1 = @static (VERSION >= v"1.0-" ? true : false)
-    if (is_julia_v1 && arg.head === :kw) || arg.head === :(=)
+    # [i=1] and x[i=1] parse as :(=) and :kw, respectively.
```

Comment:
Can we keep the `getindex(x, i=1)` ? Just saying it is parsed as `:kw` is less helpful

Reply:
Done